### PR TITLE
[release-ocm-2.4] Bug 2033463: Rename the UI deployment template (#3056)

### PR DESCRIPTION
**Pull request by lranjbar**
# Assisted Pull Request

## Description

This is a backport of the PR #3056 to release-ocm-2.4

## List all the issues related to this PR

- [ ] New Feature
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [X] CI/CD 

## What environments does this code impact?

- [X] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @osherdp 
/cc @eliorerz 

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -364,10 +364,11 @@ deploy-onprem:
...
-	podman run -dt --pod assisted-installer --env-file onprem-environment --pull always -v $(PWD)/deploy/ui/nginx.conf:/opt/bitnami/nginx/conf/server_blocks/nginx.conf:z --name ui quay.io/ocpmetal/ocp-metal-ui:latest
-	podman run -dt --pod assisted-installer --env-file onprem-environment ${PODMAN_PULL_FLAG} --env DUMMY_IGNITION=$(DUMMY_IGNITION) \
-		--restart always --name installer $(SERVICE)
-	./hack/retry.sh 90 2 "curl http://127.0.0.1:8090/ready"
+	podman run -dt --pod assisted-installer --env-file onprem-environment --pull always -v $(PWD)/deploy/ui/nginx.conf:/opt/bitnami/nginx/conf/server_blocks/nginx.conf:z --name ui quay.io/edge-infrastructure/assisted-installer-ui:latest
+	podman run -dt --pod assisted-installer --env-file onprem-environment --pull always --restart always --name image-service quay.io/edge-infrastructure/assisted-image-service:latest
+	podman run -dt --pod assisted-installer --env-file onprem-environment ${PODMAN_PULL_FLAG} --env DUMMY_IGNITION=$(DUMMY_IGNITION) --restart always --name installer $(SERVICE)
+	./hack/retry.sh 90 2 "curl -f http://127.0.0.1:8090/ready"
+	./hack/retry.sh 60 10 "curl -f http://127.0.0.1:8888/health"
```
Comment (osherdp) [inline Makefile:371]: nit: this seems unrelated to the original PR
although I don't think it will do any harm

Comment (lranjbar) [reply Makefile:371]: This was the merge conflict in the Makefile

**Review by osherdp — APPROVED** [review]

**Review by eliorerz — APPROVED** [review]
LGTM
